experimental/trusted_shuffler/scripts/run

```diff
@@ -2,15 +2,22 @@
 
 readonly SCRIPTS_DIR="$(dirname "$0")"
 
+cargo build --manifest-path=experimental/trusted_shuffler/server/Cargo.toml
 cargo build --manifest-path=experimental/trusted_shuffler/backend/Cargo.toml
- "${SCRIPTS_DIR}/run_backend" &
-readonly BACKEND_PID=$!
+cargo build --manifest-path=experimental/trusted_shuffler/client/Cargo.toml
 
-cargo build --manifest-path=experimental/trusted_shuffler/server/Cargo.toml
-"${SCRIPTS_DIR}/run_server" &
+RUST_LOG=info cargo run --manifest-path=experimental/trusted_shuffler/server/Cargo.toml -- \
+  --k=1 --listen-address="[::]:8080" --backend-url="http://localhost:8888" &
+sleep 1
```

Comment:
Why do we sleep?

Reply:
Otherwise the background task may run after the client and thus fail

Reply:
I assumed something like that. :) Maybe we want to add a comment?

Reply:
![41z0HDnxBGL._AC_SY580_.jpg](https://user-images.githubusercontent.com/387628/165943124-ec2ffd33-285b-4dfe-9eb7-3b546c82df7a.jpg)

Reply:
It's the ultimate Bash synchronization primitive